CMakeLists.txt

```diff
@@ -48,7 +48,13 @@ if (CMAKE_C_COMPILER_ID MATCHES "GNU")
 endif ()
 
 set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -DDEBUG")
-set(LIBS -lxenstore)
+
+find_package(XcpNgGeneric REQUIRED)
+
+set(LIBS 
+  -lxenstore
```

Comment:
It will be nice to write a `FindXenstore.cmake` in the future. You can add a TODO here. I known it's not related to this PR.

Reply:
:+1: